core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityComponent.scala

```diff
@@ -80,6 +81,22 @@ class CompactEntityQuery(driverComponent: DriverComponent) extends RawSqlQuery w
     uniqueResult(selectStatement.as[CompactEntityRecord])
   }
 
+  /**
+   * Get all entities of the given type in workspace
+   *
+   * `execution plan: simple select; indexed by idx_entity_type_name; using where`
```

Comment:
I was really unsure how to characterize and summarize the execution plans, i picked out what seemed to be the key words modeled on previous execution plan comments.  Please let me know if these make sense/are helpful

Reply:
+1 - the real purpose of documenting the execution plans is to ensure that we've looked at them and are ok with them; it's less about leaving documentation for the future